packages/core/src/modules/application/use-cases/review-application.ts

```diff
@@ -24,6 +24,7 @@ export async function reviewApplication({
       'applications.schoolId',
     ])
     .where('id', '=', applicationId)
+    .where('status', '=', ApplicationStatus.PENDING)
     .executeTakeFirstOrThrow();
 
```

Comment:
Previously we'd throw an error if the application wasn't found. But now that we're also checking for the pending status, I don't think we should throw an error per se since we know about this case now.

Let's use `.executeTakeFirst` instead of `.executeTakeFirstOrThrow` and just return early if the application isn't found, ie:

```
if (!application) {
  return;
}
```

Reply:
I'm going to bring this home - just updated!